guava/src/com/google/common/base/package-info.java

```diff
@@ -20,40 +20,40 @@
  *
  * <h2>Contents</h2>
  *
- * <h3>String-related utilities</h3>
+ * The classes in this package that are most commonly useful are:
+ *
+ * <h3>String utilities</h3>
  *
  * <ul>
- *   <li>{@link com.google.common.base.Ascii}
- *   <li>{@link com.google.common.base.CaseFormat}
- *   <li>{@link com.google.common.base.CharMatcher}
- *   <li>{@link com.google.common.base.Charsets}
- *   <li>{@link com.google.common.base.Joiner}
- *   <li>{@link com.google.common.base.Splitter}
- *   <li>{@link com.google.common.base.Strings}
+ *   <li>{@link Ascii}
+ *   <li>{@link CaseFormat}
+ *   <li>{@link CharMatcher}
+ *   <li>{@link Splitter}
+ *   <li>{@link Strings}
  * </ul>
  *
  * <h3>Function types</h3>
  *
  * <ul>
- *   <li>{@link com.google.common.base.Function}, {@link com.google.common.base.Functions}
- *   <li>{@link com.google.common.base.Predicate}, {@link com.google.common.base.Predicates}
```

Comment:
I think the `Functions` and `Predicates` classes are still worth mentioning.

Reply:
Thanks. Do you have any specific methods that you'd call out as important nowadays? I'm expecting for most to be replaceable with method references or JDK APIs nowadays (e.g., `Map::get` (with caveats), `Objects::nonNull`). And I want to document more of those—but I haven't looked yet, so you may be able to save me some time :)

Reply:
I thought `Functions` had more than it does - ignore my comment above regarding it. When it comes to`Predicates`, though:
- `Predicates.not()`: really useful for cases like `not(t1 -> ...)`
- `Predicates.alwaysFalse()` and `Predicates.alwaysTrue()`: handy like `Function.identity()`

Reply:
Thanks. Java added [`Predicate.not`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/function/Predicate.html#not(java.util.function.Predicate)) in Java 11, and we generally recommend `x -> true` / `x -> false` over the `Predicates.always*` methods (similar to how we (and Effective Java) recommend `v -> v` over `Function.identity())`. (Hmm, actually, we _don't_ recommend `v -> v` yet in our Javadoc, just in some internal doc we have.... I'll take care of that.)